packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBtcAddresses/selectors.js

```diff
@@ -75,7 +84,15 @@ export const getData = (state, ownProps) => {
             .map(toDropdown)
             .map(toGroup('Lockbox'))
     ]).map(([b1, b2, b3]) => {
-      const data = reduce(concat, [], [b1, b2, b3])
+      const importedAddressesSorted = set(
```

Comment:
Did you try doing something on line 77 instead?

.map(toDropdown)
.map(sortDropdown)

?